harness/determined/cli/tensorboard.py

```diff
@@ -30,45 +30,35 @@ def start_tensorboard(args: Namespace) -> None:
     )
 
     resp = bindings.post_LaunchTensorboard(cli.setup_session(args), body=body)
+    tsb = resp.tensorboard
 
     if args.detach:
         print(resp.tensorboard.id)
         return
 
+    render.report_job_launched("tensorboard", tsb.id)
+
     if resp.warnings:
         cli.print_warnings(resp.warnings)
     currentSlotsExceeded = (resp.warnings is not None) and (
         bindings.v1LaunchWarning.CURRENT_SLOTS_EXCEEDED in resp.warnings
     )
-    url = "tensorboard/{}/events".format(resp.tensorboard.id)
-    with api.ws(args.master, url) as ws:
-        for msg in ws:
-            if msg["log_event"] is not None:
-                # TensorBoard will print a url by default. The URL is incorrect since
-                # TensorBoard is not aware of the master proxy address it is assigned.
-                if "http" in msg["log_event"]:
-                    continue
-
-            if msg["service_ready_event"] and resp.tensorboard.serviceAddress is not None:
-                if args.no_browser:
-                    url = api.make_url(args.master, resp.tensorboard.serviceAddress)
-                else:
-                    url = api.browser_open(
-                        args.master,
-                        request.make_interactive_task_url(
-                            task_id=resp.tensorboard.id,
-                            service_address=resp.tensorboard.serviceAddress,
-                            resource_pool=resp.tensorboard.resourcePool,
-                            description=resp.tensorboard.description,
-                            task_type="tensorboard",
-                            currentSlotsExceeded=currentSlotsExceeded,
-                        ),
-                    )
-
-                print(colored("TensorBoard is running at: {}".format(url), "green"))
-                command.render_event_stream(msg)
-                break
-            command.render_event_stream(msg)
+    cli.wait_ntsc_ready(cli.setup_session(args), api.NTSC_Kind.tensorboard, tsb.id)
```

Comment:
hmm this seems expected. are there any specific line you're worried about

Reply:
Oh, should've looked closer because this isn't an issue but a nit
More specifically this line:
`Waiting stopped: task has been terminated.            `                
imo it seems redundant if  `task has been terminated.` will be repeated again.

Reply:
it's an artifact of the loading animation I put in. it takes more work to make it more flexible. something we can do separately later if it becomes too annoying.